HW04/Nastja_Kireeva/lib/mentor.rb

```diff
@@ -0,0 +1,38 @@
+class Mentor
+  attr_reader :homeworks, :fullname, :students, :notifications
+  attr_accessor :student
+
+  def initialize(fullname:)
+    @fullname = fullname
+    @homeworks = []
+    @notifications = []
+    @students = []
+  end
+
+  def add_homework(title:, description:, student:)
+    homework = Homework.new(title: title, description: description, student: student, mentor: self)
+    homeworks << homework
+    notification = Notification.new(homework: homework).send_message[:new]
```

Comment:
Can we change this interface a little bit? 🤔 
```suggestion
    homeworks << homework
    notification = Notification.new(homework: homework, status: :new).message
    
    # notification.rb
    def message
      { ... }[status]
    end
```